src/main/cc/math/noise_parameters_computation.cc

```diff
@@ -30,19 +33,40 @@ int ComputateMuPolya(double epsilon, double delta, int sensitivity, int n) {
 
 }  // namespace
 
-math::DistributedGeometricRandomComponentOptions GetPublisherNoiseOptions(
+DistributedGeometricNoiseComponentOptions GetGeometricPublisherNoiseOptions(
     const wfa::any_sketch::DifferentialPrivacyParams& params,
     int publisher_count) {
   ABSL_ASSERT(publisher_count > 0);
   double success_ratio = std::exp(-params.epsilon() / publisher_count);
   int offset =
       ComputateMuPolya(params.epsilon(), params.delta(), publisher_count, 1);
   return {
-      .num = 1,
+      .contributor_count = 1,
       .p = success_ratio,
       .truncate_threshold = offset,
       .shift_offset = offset,
   };
 }
 
+DistributedDiscreteGaussianNoiseComponentOptions
+GetDiscreteGaussianPublisherNoiseOptions(
+    const wfa::any_sketch::DifferentialPrivacyParams& params) {
+  double epsilon = params.epsilon();
+  double delta = params.delta();
+
+  ABSL_ASSERT(epsilon > 0);
+  ABSL_ASSERT(delta > 0);
+
+  // TODO(@iverson52000): Update formula to sigma = 1 / sqrt(2 * rho) once
+  // Almost Concentrated DP (ACDP) for accounting is implemented
+
+  // The sigma calculation formula is a closed-form formula from The Algorithmic
```

Comment:
Strictly speaking, this formula is only for *continuous* Gaussian noise and I'm not aware of any place in literature that has this formula (or a similar one) for *discrete* Gaussian. I think it is ok to use as an approximation in the short run since we are anyway moving to ACDP, but I think we should note the downside here for clarity.